portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,38 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import java.util.ArrayList;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Juan!");
+    ArrayList<Comment> jsonarray = new ArrayList<Comment>();
+    Comment newcomment1 = new Comment("I love this site.");
+    Comment newcomment2 = new Comment("This is so cool.");
+    Comment newcomment3 = new Comment("All the cool kids are coding nowadays.");
+    jsonarray.add(newcomment1);
+    jsonarray.add(newcomment2);
+    jsonarray.add(newcomment3);
+
+    String json = convertToJson(jsonarray);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+
+  }
+
+  private String convertToJson(ArrayList<Comment> alist) {
```

Comment:
nit: rename alist to comments/commentsList to make it clear what the variable contains